### PR TITLE
feat: [#186084930] Hide landing page presentational images from screen readers

### PR DESCRIPTION
**Pull request by seidior**
## Description

Removal of alternative text on the Navigator landing page for eligible decorative images, and amendment of alternative text for eligible informative images.

This does not include any changes to the "Made with ❤️ by the" text, as that will be covered in a different pull request.

### Ticket

[#186084930](https://www.pivotaltracker.com/story/show/186084930)

### Approach

Cues were taken from [W3C's guide on Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/), specifically with regard to `alt=""` and `role="presentation"`. The decision matrix used was:

- If image was both decorative and illustrative in nature, it got both `alt=""` and `role="presentation"`.
- If image was decorative, it got `alt=""`.
- If image was informative, its alternative text was updated.

### Steps to Test

Verify that the alt text matches the revisions in [the associated Figma file](https://www.figma.com/file/UjmQDBkTbNfv8Kiyi8178b/My-Account-Homepage-images-annotation-16-Oct-2023-%28Copy%29?type=design&node-id=0%3A1&mode=design&t=72D41Yoa57rlhn11-1).

### Notes

Nothing was done to the social images in the footer; in order to follow [guidelines](https://css-tricks.com/accessible-svgs/#aa-2-inline-svg) to update these images, a pull request would need to be opened against [USWDS](https://github.com/uswds/uswds) to modify the [base images](https://github.com/uswds/uswds/blob/develop/packages/usa-icon/src/img/uswds-icons/github.svg?short_path=09f6740), then those changes would need to trickle down to this repository.

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation (EX: [Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po)), if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts`

### CODE REVIEW

File: web/src/components/njwds/Hero.tsx
Comment (adidner) [inline web/src/components/njwds/Hero.tsx:84]: Is there a reason this didn't get the role presentation as well?

File: web/src/components/njwds/Hero.tsx
Comment (adidner) [inline web/src/components/njwds/Hero.tsx:155]: Same here

**Review by jphechter — APPROVED** [review]
I think this all works. Talked with Spencer about this. Some of the images (such as business owners picture) are not getting alt text after back and forth with content/ux.

LGTM 👍

**Review by adidner — APPROVED** [review]
2 questions otherwise looks good.

File: web/src/components/njwds/Hero.tsx
Comment (seidior) [reply web/src/components/njwds/Hero.tsx:84]: These were the guidelines that I followed:

- If image was both decorative and illustrative in nature, it got both alt="" and role="presentation".
- If image was decorative, it got alt="".
- If image was informative, its alternative text was updated.

That was my read from the W3C page on this, as the only example that was given for role="presentation" was illustrative in nature; the rest did not mention that. I'm open to other interpretations though: if you think all should get role="presentation", I'll defer to your expertise!

**Review by somabadri — APPROVED** [review]
LGTM, after address

**Pull request merged**
